calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
I'm sorry to throw more spanners in, but "disable the use of Calico profiles" is unclear: I'm not sure what that actually means.

I think the best way to go is to say something about the effect - will traffic be allowed at all?  If not, maybe say something like "when enabled traffic must be explicitly allowed by configuring Calico policies.".  People might not know how the libnetwork networks turn into profiles, but they need to know what happens to traffic.